Final status:
Process still alive after running test suite
stdout and stderr:
DEBUG:root:[*] running method core_negotiate_tlv_encryption
DEBUG:root:[*] Negotiating TLV encryption
DEBUG:root:[*] RSA key: 30820122300d06092a864886f70d01010105000382010f003082010a0282010100a656a81ce6976813581bd21442d2408492edff4ae0d70c25c9b0de0a568bb763bc2e4b05848bf201cbe2d8ef163d73fade4d3e9b1ad20e796471bf2d68c243c601dac9c57de672c6192df501a0f2b25bfb62c033f0bea8fd83d578fdec4ad978aa44a1d59f0ca5457a8c471b7a32f2003c270cb7d25ff7d17a0a0abb6a97613d6d341bd085c4d0852eb75a3674e240b465b188d19665cf4d4727b77f3c0daf41e679913877f12fe836e47ba4bc7ba3463af57c39b15e30e27bc2f1809d678b9a5560e0e49df570a58b95a04c46e0ca3afff18c1d935be65a08fa78feb403591390a79a59984e784a4cf949c51d39ab72ac4a3e9a4e8d306b51902209044ae7af0203010001
DEBUG:root:[*] AES key: 0x95230662ad3e8ac1bd7e2b49841f6d4ecb00d5f16fd01741e577b384e353bfe6
DEBUG:root:[*] Encrypted AES key: 0x2c3d53820b99b4bc8ec039b94dac383c00dfc87dceb8a2eb53d3d859723cf9110ed1585dd998916d422fd132f1e28eb370d4f320c71cc816fbf33498868456af717472df56ddc2c72ffd36decfabff55a5ca0a8302ea41c18127b1007d3172340b652d97557da03f3c8396b093b39c5fd6ed8c0148aef2c4b05bb0ae42f9ae20bc1324e396428e5fd5db8c78e35bd0bb59107f4220bb3aec67eff3ee2109c538aa439bae275f8db09bc931f71f730151cdd4ad7378331a5f0836a07cf970552a7cad18fbdb52048f832bf833706659e42be58fc648f618cc5ab8ae573134ccff04ef7f5e7f74911e217d6395925c7e47c4d988d1b911fa5fee904462f4522f99
DEBUG:root:[*] TLV encryption sorted
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_set_session_guid
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_enumextcmd
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_enumextcmd
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_loadlib
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_getwd
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getuid
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_sysinfo
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_set_uuid
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_net_config_get_interfaces
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_net_config_get_routes
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_getwd
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_separator
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_chdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_getwd
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir-teosdigb'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_mkdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_chdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_getwd
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_getwd
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'c:\\'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_mkdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 1 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 1
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir/directory'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_mkdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 2 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 2
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_ls
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_dir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_process_execute
DEBUG:root:[*] starting process: ['/bin/sh', '-c', 'command -v ln || which ln && echo FZTYCZBE']
DEBUG:root:[*] added process id: 3844, handle: 1
DEBUG:root:[*] added channel id: 3 type: MeterpreterProcess
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[-] method core_channel_read resulted in error: #1
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_process_close
DEBUG:root:[*] closed and removed process handle: 1
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_mkdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 4 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 4
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_process_execute
DEBUG:root:[*] starting process: ['/bin/sh', '-c', 'ln -s $(pwd)/meterpreter-test-dir $(pwd)/meterpreter-test-dir.lnk']
DEBUG:root:[*] added process id: 3845, handle: 2
DEBUG:root:[*] added channel id: 5 type: MeterpreterProcess
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_process_close
DEBUG:root:[*] closed and removed process handle: 2
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_dir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir.lnk'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_dir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir.1'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_mkdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir.2'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_mkdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 6 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 6
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_process_execute
DEBUG:root:[*] starting process: ['/bin/sh', '-c', 'ln -s $(pwd)/meterpreter-test-dir.1 $(pwd)/meterpreter-test-dir.2/link']
DEBUG:root:[*] added process id: 3860, handle: 3
DEBUG:root:[*] added channel id: 7 type: MeterpreterProcess
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_process_close
DEBUG:root:[*] closed and removed process handle: 3
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_dir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_dir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'c:\\boot.ini'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'c:\\pagefile.sys'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: '%WINDIR%\\system32\\notepad.exe'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: '%WINDIR%\\system32\\calc.exe'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 8 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 8
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 9 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_eof
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 9
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 10 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 10
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 11 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_eof
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 11
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 12 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 12
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 13 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_eof
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 13
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_file
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-moved'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 14 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 14
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-moved'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_file_move
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_file
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 15 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 15
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 16 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_eof
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_eof
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 16
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_file
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 17 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 17
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 18 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 18
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 19 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_eof
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 19
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_file
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_chdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_getwd
DEBUG:root:[*] sending response packet
